.github/ISSUE_TEMPLATE/Service_team_request.yaml

```
@@ -0,0 +1,75 @@
+name: "[Microsoft Internal / Service Team] Feature Request"
+description: Use this template to create a feature request for microsoft internal or service team
+body:
+  - type: markdown
+    id: example
+    attributes:
+      value: |
+        [How to Request a Feature in CLI](https://github.com/Azure/azure-cli/blob/dev/doc/how_to_request_a_feature_in_cli.md)
+        [A Great Example](https://github.com/Azure/azure-cli/issues/22275)
+  - type: textarea
+    id: command
+    attributes:
+      label: Related command
+      description: Please provide the related command with az {command} if you can, so that we can quickly route to the related team for help.
+      placeholder: e.g., `az vm create`
```

Another small suggestion is that some feature requests are mainly related to parameters. If they also provide parameter information, it can help us evaluate requirements faster
```suggestion
      description: Please provide the related commands and parameters with az {command} if you can, so that we can quickly route to the related team for help.
      placeholder: e.g., `az vm create --image`
```